events/listener.go

```diff
@@ -169,16 +184,21 @@ func (router *EventRouter) subscribeToEvents(subscribeURL string, accessKey stri
 	ws, resp, err := dialer.Dial(subscribeURL, headers)
 
 	if err != nil {
-		defer resp.Body.Close()
-		body, _ := ioutil.ReadAll(resp.Body)
 		log.WithFields(log.Fields{
-			"status":          resp.Status,
-			"statusCode":      resp.StatusCode,
-			"responseHeaders": resp.Header,
-			"responseBody":    string(body[:]),
-			"error":           err,
-			"subscribeUrl":    subscribeURL,
-		}).Error("Failed to subscribe to events.")
+			"subscribeUrl": subscribeURL,
+		}).Errorf("Error subscribing to events: %s", err)
+		if resp != nil {
+			log.WithFields(log.Fields{
```

Comment:
FYI, I've come to dislike log.WithFields and prefer just normal <level>F log methods. So, in the future, feel free to use those, don't need to keep with the bad precedent i set.

No need to change it in this this PR now though.